packages/mds-utils/utils.ts

```diff
@@ -507,14 +497,42 @@ function getPolygon(geographies: Geography[], geography: string): Geometry | Fea
   return res.geography_json
 }
 
+function areThereCommonElements<T>(arr1: T[], arr2: T[]) {
+  const set = new Set([...arr1, ...arr2])
+  return set.size !== arr1.length + arr2.length
+}
```

Comment:
I may have commented this already, but we could use two different generics for this. We don't _necessarily_ know that our inputs are of the same type, so we might want to do
```suggestion
function areThereCommonElements<T, U>(arr1: T[], arr2: U[]) {
  const set = new Set([...arr1, ...arr2])
  return set.size !== arr1.length + arr2.length
}
```